### PR TITLE
10 Racial Discrimination

### PR DESCRIPTION
**Pull request by JCardenasRdz**
Project 10 is done

### CODE REVIEW

File: Projects/10_EDA_Racial_Discrimination/10-Racial_Discrimination.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    Con_Table.columns = ['No','Yes']
+    Con_Table.index = ['Black','White']
+    return Con_Table, df
+
+def my_chi2(X):
+    '''
+    chi2, p, _, expected_distribution = scipy.stats.chi2_contingency(X,correction=True)
+    '''
+    chi2, p, _, expected_distribution = scipy.stats.chi2_contingency(X,correction=True)
+    print('The p-value is : '+str(p))
```
Comment (amirziai) [inline Projects/10_EDA_Racial_Discrimination/10-Racial_Discrimination.py:22]: Considering using this format: 
`The p-value is: {}.format(p)`

More details here: https://pyformat.info/

File: Projects/10_EDA_Racial_Discrimination/10-Racial_Discrimination.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    Con_Table = pd.crosstab(df.race,df.call)
+    Con_Table.columns = ['No','Yes']
+    Con_Table.index = ['Black','White']
+    return Con_Table, df
+
+def my_chi2(X):
+    '''
+    chi2, p, _, expected_distribution = scipy.stats.chi2_contingency(X,correction=True)
+    '''
+    chi2, p, _, expected_distribution = scipy.stats.chi2_contingency(X,correction=True)
```
Comment (amirziai) [inline Projects/10_EDA_Racial_Discrimination/10-Racial_Discrimination.py:21]: Space

File: Projects/10_EDA_Racial_Discrimination/10-Racial_Discrimination.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    print('The p-value is : '+str(p))
+    print('The test statistic is : '+str(chi2))
+    print('The expected distribution is :')
+    print(round(expected_distribution))
+    
+def prepare_for_classification(DF):
```
Comment (amirziai) [inline Projects/10_EDA_Racial_Discrimination/10-Racial_Discrimination.py:27]: lower_case variable name

File: Projects/10_EDA_Racial_Discrimination/10-Racial_Discrimination.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    Y = call 
+    '''
+    df = DF[['yearsexp','sex','education','race','manager']].copy()
+    df['sex'][df['sex']=='f']=1.0
+    df['sex'][df['sex']=='m']=0.0
+    df['race'][df['race']=='w']=0.0
+    df['race'][df['race']=='b']=1.0
+    X = df.values
+    y = DF.call.values
+    df['Call'] = DF.call
```
Comment (amirziai) [inline Projects/10_EDA_Racial_Discrimination/10-Racial_Discrimination.py:39]: Why is this line necessary?

File: Projects/10_EDA_Racial_Discrimination/10-Racial_Discrimination.py (hunk 1 above)
Comment (JCardenasRdz) [reply Projects/10_EDA_Racial_Discrimination/10-Racial_Discrimination.py:22]: very cool. I used:
~~~Python
print('The p-value is : '+ '{:.3}'.format(p) )
print('The test statistic is : '+ '{:.3}'.format(chi2) )
~~~

File: Projects/10_EDA_Racial_Discrimination/10-Racial_Discrimination.py (hunk 4 above)
Comment (JCardenasRdz) [reply Projects/10_EDA_Racial_Discrimination/10-Racial_Discrimination.py:39]: Is not. I deleted it

File: Projects/10_EDA_Racial_Discrimination/10-Racial_Discrimination.py (hunk 1 above)
Comment (amirziai) [reply Projects/10_EDA_Racial_Discrimination/10-Racial_Discrimination.py:22]: This is considered more idiomatic:
```python
print('The p-value is : {:.3f}'.format(p))
```
